src/opcode-code-table.c

```diff
@@ -38,4 +38,8 @@ uint32_t WabtOpcodeCodeTable[WABT_OPCODE_CODE_TABLE_SIZE] = {
   [(prefix << 8) + code] = Name,
 #include "opcode.def"
 #undef WABT_OPCODE
+
+  // TODO(binji): LLVM still generates 0xfd03 for v8x16.shuffle, see
+  // https://github.com/WebAssembly/wabt/issues/1339
+  [(0xfd << 8) + 0x03] = V8X16Shuffle,
```

Comment:
I tried this change over in #1339 but when I use it to decode old-numbered shuffles with ` make && bin/wasm-objdump -d --debug ../old.wasm` I get:
```
...
on_opcode: 0x3c82: v8x16.shuffle
warning: 0x3c82 missing opcode callback at 0x3c81 (0xfd=<invalid>)
```
This is what led me to believe that there might be a change needed somewhere else?

Reply:
Ah, I see what you mean. It's annoying because the new encoding is 3 bytes and the old encoding is 2 bytes. So it doesn't alias quite right. Not sure the least hacky way to fix this... :-)

Reply:
Sounds like the new renumbering should be happening soon -- perhaps the simplest solution is for you to recompile wabt with the old numbering. Sorry, but it seems like it's not quite as easy as I'd hope to add it in!

Reply:
I see #1342... thanks! But am I confused? I thought both encodings were 2 bytes long?

Reply:
The prefix byte is a single byte, but the opcode after that is LEB128 encoded, so numbers above 0x7f take more than one byte to encode.